configuration/drugs/drugs.csv

```diff
@@ -56,15 +58,18 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
 ,,Paracetamol 200mg Tablet,Cambodia_Paracetamol,Tablet,200mg,
 ,,Paracetamol 325mg Tablet,Cambodia_Paracetamol,Tablet,325mg,
 ,,Paracetamol 100mg Tablet,Cambodia_Paracetamol,Tablet,100mg,
-,,"Penicillin V 1,000,000IU Tablet",Cambodia_Penicillin V,Tablet,"1,000,000IU",
+,,"Penicillin V 1,000,000iu Tablet",Cambodia_Penicillin V,Tablet,"1,000,000iu",
 ,,Perindopril 5mg Tablet,Cambodia_Perindopril,Tablet,5mg,
 ,,Phenobarbital 100mg Tablet,Cambodia_Phenobarbital,Tablet,100mg,
 ,,Praziquantel 600mg Tablet,Cambodia_Praziquantel,Tablet,600mg,
 ,,Prednisolone 5mg Tablet,Cambodia_Prednisolone,Tablet,5mg,
+,,Ranitidine 300mg Tablet,Cambodia_Ranitidine,Tablet,300mg,
 ,,Salbutamole 2mg Tablet,Cambodia_Salbutamole,Tablet,2mg,
 ,,Salbutamole 100mcg/actuation Inhaler,Cambodia_Salbutamole,Inhaler,100mcg/actuation,
+,,Salbutamole 2.5mg/2.5ml Inhaler,Cambodia_Salbutamole,Inhaler,2.5mg/2.5ml,
 ,,Spironolactone 25mg Capsule,Cambodia_Spironolactone,Capsule,25mg,
-,,"Vitamine A 200,000IU Tablet",Cambodia_Vitamine A,Tablet,"200,000IU",
+,,Spironolactone 50mg Capsule,Cambodia_Spironolactone,Capsule,50mg,
+,,"Vitamine A 200,000iu Tablet",Cambodia_Vitamine A,Tablet,"200,000iu",

```

Comment:
Same here as well (see remarks above). FSN is renamed. New concept will be created.
Just find the UUID in the prod server and add it to this drug line.

Reply:
I changed these 3 to match the changed name already in prod server, so is this necessary?

Reply:
@ouiliam that's great if you have changed it on prod already, so then no that is not necessary to do add the UUID.